context/app/static/js/components/cells/DatasetsTable/DatasetsTable.jsx

```diff
@@ -1,11 +1,14 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useRef } from 'react';
 import Table from '@material-ui/core/Table';
 import TableHead from '@material-ui/core/TableHead';
 import TableBody from '@material-ui/core/TableBody';
 import TableRow from '@material-ui/core/TableRow';
 import TableCell from '@material-ui/core/TableCell';
+import { useTransition, animated, config } from 'react-spring';
```

Comment:
Next time we talk, if you can explain to me what the limits of CSS transitions are, that would be helpful. A little leery of another dependency if it could be handled on a lower level.